api/client/utils.go

```diff
@@ -343,3 +344,19 @@ func readBody(stream io.ReadCloser, statusCode int, err error) ([]byte, int, err
 	}
 	return body, statusCode, nil
 }
+
+// getTimestamp tries to parse given string as RFC3339 time
+// or Unix timestamp, if successful returns a Unix timestamp
+// as string otherwise returns value back.
+func getTimestamp(value string) string {
```

Comment:
yeah I guess we could move, this used to be buried in the code here, I just extracted into a method.